class-pikpay.php

```diff
@@ -567,13 +573,13 @@ public function generate_form($order_id)
             $currency = $order_meta["_order_currency"][0];
         }
 
-        //Convert currency to match PikPay's requested currency
+        //Convert currency to match Monri's requested currency
         if ($currency == "KM") {
             $currency = "BAM";
         }
 
         //Generate digest key
-        $digest = hash('sha512', $this->pikpaykey . $order->get_id() . $order_total . $currency);
+        $digest = hash('sha512', $this->monri_key . $order->get_id() . $order_total . $currency);
```

Comment:
@4050350194 while you are at it - rename `monri_key` to `monri_merchant_key`.

Reply:
Resolved by commit ID: 2ecebc1d36ff40711be89c5071d93220744b53a9